base/inference.jl

```diff
@@ -1877,6 +1877,26 @@ function finish(me::InferenceState)
     # finalize and record the linfo result
     me.inferred = true
 
+    # determine and cache inlineability
+    inlineable = false
+    if isdefined(me.linfo, :def)
+        cost = 1000
+        if me.linfo.def.module === _topmod(me)
+            name = me.linfo.def.name
+            if me.linfo.isva && (name === :+ || name === :* || name === :min || name === :max)
+                inlineable = true
```

Comment:
maybe just `cost ÷= 20`? i think the previous test seemed very strict on which method it was matching to hopefully avoid the corner cases where inlining would make it slower.